Source/WebCore/html/HTMLAnchorElement.cpp

```diff
@@ -362,10 +362,11 @@ void HTMLAnchorElement::sendPings(const URL& destinationURL)
     if (!document().frame())
         return;
 
-    if (!hasAttributeWithoutSynchronization(pingAttr) || !document().settings().hyperlinkAuditingEnabled())
+    const auto& pingValue = attributeWithoutSynchronization(pingAttr);
+    if (pingValue.isNull() || !document().settings().hyperlinkAuditingEnabled())
```

Comment:
Ideally, we'd do the hyperlinkAuditingEnabled check earlier to avoid even looking up the attribute when the setting is disabled.

Reply:
Should I add early return before const as below?
```
if (!document().settings().hyperlinkAuditingEnabled())
  return;
```

Reply:
Yes please.